terms.html

```diff
@@ -108,7 +108,7 @@
   </dd>
   <dt><dfn data-lt="party|parties|party's">party</dfn></dt>
   <dd>
-An <a>entity</a> that has 'a mind of its own', which means that, amongst others, that it autonomously (sovereignly)
+An <a>entity</a> that has 'a mind of its own', which means, amongst others, that it autonomously (sovereignly)
```

Comment:
I can live with `amongst` though it sounds odd to me; `others` should still be `other things`, as suggested previously.
```suggestion
An <a>entity</a> that has 'a mind of its own', which means, amongst other things, that it autonomously (sovereignly)
```